pkg/resources/fluentbit/config.go

```diff
@@ -35,7 +35,7 @@ var fluentBitConfigTemplate = `
     Coro_Stack_Size    {{ .CoroStackSize }}
     {{- if .Monitor.Enabled }}
     HTTP_Server  On
-    HTTP_Listen  0.0.0.0
```

Comment:
another way: 

```
HTTP_Listen  [::]
```

works on IPv6 and doesn't need the Statefulset patch

Reply:
I mean the addional env var on $POD_IP in the Statefulset manifest. But this is not required anymore. Tested on Ipv6 and Ipv4 cluster the definition of `[::]` works.
The end2end test failed of the change in Statefulset manifest, but this should fixed now.